ui_testing/testcases/basic_tests/test04_testunits.py

```diff
@@ -812,3 +812,102 @@ def test025_edit_approach_overview_button(self):
         self.assertEqual(self.base_selenium.get_url(), '{}testUnits'.format(self.base_selenium.url))
         self.base_selenium.LOGGER.info('clicking on Overview confirmed')
 
+    def test_028_changing_testunit_type_update_fields_accordingly(self):
```

Comment:
parameterized it

Reply:
![image](https://user-images.githubusercontent.com/44477333/71586460-f951ee00-2b22-11ea-9512-8613f479baa8.png)